test/integration/collection-management/collection_management.spec.test.js

```diff
@@ -3,6 +3,9 @@
 const { loadSpecTests } = require('../../spec/index');
 const { runUnifiedSuite } = require('../../tools/unified-spec-runner/runner');
 
+// The Node driver does not have a Collection.modifyCollection helper.
+const SKIPPED_TESTS = ['modifyCollection to changeStreamPreAndPostImages enabled'];
```

Comment:
Is this where you wanted the skip reason? (I think we can make a separate ticket to update the unified runner to allow passing them in.)

Reply:
Yup, this is where I was wondering about the skip reason.  I can create a follow up ticket